src/mavlink-router/mainloop.h

```diff
@@ -76,6 +81,10 @@ class Mainloop {
     int g_tcp_fd = -1;
     LogEndpoint *_log_endpoint = nullptr;
 
+    std::map<std::string, std::string> pipe_commands;
```

Comment:
Should these have a `_` prefix?

Reply:
Changed this and other new private members to have underscore.